docs/src/main/asciidoc/deploying-to-google-cloud.adoc

```diff
@@ -60,14 +60,14 @@ Then, you will need to create a `src/main/appengine/app.yaml` file, let's keep i
 
 [source, yaml]
 ----
-runtime: java11
+runtime: java17
 ----
 
 This will create a default service for your App Engine application.
 
 [NOTE]
 ====
-You can also use the new Java 17 runtime by defining `runtime: java17` instead.
+You can also use another Java runtime supported by App Engine, for example, for Java 21, use `runtime: java21` instead.
```

Comment:
I think we could probably switch that and make `java21` the default and explain how to configure `java17` manually.
Not pressing but next time you're around.

While Java 17 is the minimum version we support, we recommend Java 21.

Reply:
Done here: https://github.com/quarkusio/quarkus/pull/44321